README.md

```diff
@@ -1,5 +1,38 @@
-# PayTonight: Payment Gateway: 
+# PayTonight: Payment Gateway:
 
-It is now available at: https://paytonight.herokuapp.com/
+## Documentation
 
-### Documentation SOON
+## How to use
+
+1. Register in API through Portal https://paytonight.herokuapp.com/
+
+2. Login and Generate API Key for app
```

Comment:
2. Login and generate API credentials, make sure to save `APP_KEY` or you will need to generate another one (notice that APP ID doesn't change so)